scripts/publish-functions.sh

```diff
@@ -63,12 +63,13 @@ for f in *; do
         queuename=${f//_/-}
         echo "About to publish function $functionname"
 
+        # shellcheck disable=SC2140
         gcloud functions deploy "$functionname" --trigger-http \
             --runtime nodejs10 \
             --region "$FUNCTION_REGION" \
             --set-env-vars DRIFT_PRO_BUCKET="$BUCKET",KEY_LOCATION="$KEY_LOCATION",KEY_RING="$KEY_RING",GCF_SHORT_FUNCTION_NAME="$functionname",PROJECT_ID="$PROJECT_ID",GCF_LOCATION="$FUNCTION_REGION",PUPPETEER_SKIP_CHROMIUM_DOWNLOAD="1"
 
-        deploy_queue $queuename
+        deploy_queue "$queuename"
         )
     fi
 done
```

Comment:
While we're at it, newline at the end?

Reply:
At the end of the file or after the `deploy_queue` line?

Reply:
End of the file -- looks good now.